vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java

```diff
@@ -94,10 +94,19 @@ public void typeText(WebElement element, String text)
      * </ol>
      * @param element element to clear, must not be {@code null}
      */
+    public void clearTextAndHide(WebElement element)
```

Comment:
it makes sense to have 2 methods in `KeyboardActions`:
- `clearText`
- `hideKeyboard`

any combinations of these methods invocations should be done in steps